frontend/device/kindle/device.lua

```diff
@@ -104,6 +126,7 @@ local Kindle = Generic:new{
     -- NOTE: We can cheat by adding a platform-specific entry here, because the only code that will check for this is here.
     isSpecialOffers = isSpecialOffers(),
     hasOTAUpdates = yes,
+    hasFastWifiStatusQuery = yes,
```

Comment:
Good morning, thank you for that especially.

Reply:
Hey, good to see you (alive) ! How are you ?
(And where were/are you living? We all got very faimiliar with Ukraine geography lately :/)

Reply:
Thank you, so-so, still migrating. Hope to appear here more frequently if I can.